packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -30,97 +37,144 @@ import AngleLeftIcon from '@patternfly/react-icons/dist/esm/icons/angle-left-ico
+
 ```ts file="MenuWithFavorites.tsx"
 ```
 
+### Filtering with search input
+
+A [search input](/components/search-input) component can be placed within `<MenuInput>` to render a search input at the top of the menu. In the following example, the `onChange` property of the text input is passed a callback that filters menu items as a user types.
+
+```ts file="MenuFilteringWithSearchInput.tsx"
+
 ### Option single select
 
+The following example demonstrates an option select menu that persists a selected menu item. Use the `selected` property on the `<Menu>` to label a selected item with a checkmark. You can also use the `isSelected` property on a `<MenuItem>` to indicate that it is selected.
+
 ```ts file="MenuOptionSingleSelect.tsx"
 ```
 
 ### Option multi select
 
+You may need to persist multiple selections that a user makes, as demonstrated below.
+
 ```ts file="MenuOptionMultiSelect.tsx"
 ```
 
 ### With drilldown
 
+Use a drilldown menu to contain different levels of menu items. When a parent menu item (an item that has a submenu of children) is selected, the menu is replaced with the children items. 
+
+- To indicate that a menu contains a drilldown, use the `containsDrilldown` property.
+- To indicate the path of drilled-in menu item ids, use the `drilldownItemPath` property. 
+- Pass in an array of drilled-in menus with the `drilledInMenus` property.
+- Use the `onDrillIn` and `onDrillOut` properties to contain callbacks for drilling into and drilling out of a submenu, respectively. 
+- To account for updated heights as menus drill in and out of use, use the `onGetMenuHeight` property. When starting from a drilled-in state, the `onGetMenuHeight` property must define the height of the root menu.
+
 ```ts file="./MenuWithDrilldown.tsx" isBeta
 ```
 
-### With drilldown - initial drill in state
+### Initially drilled-in menu
 
-To render an initially drilled in menu, the `menuDrilledIn`, `drilldownPath`, and `activeMenu` states must be set to an initial state. The `menuHeights` state must also be set, defining the height of the root menu. The `setHeight` function passed into the `onGetMenuHeight` property must also account for updating heights, other than the root menu, as menus drill in and out of view.
+To render an initially drilled-in menu, the `drilldownItemPath`, `drilledInMenus`, and `activeMenu` properties must be set to default values.
 
 ```ts file="./MenuWithDrilldownInitialState.tsx" isBeta
 ```
 
 ### With drilldown - submenu functions
 
+For added flexibility with large menus, you may create a menu by passing a function to `drilldownMenu`. This approach allows you to create menu items dynamically, rather than creating everything up front.
+
 ```ts file="./MenuWithDrilldownSubmenuFunctions.tsx" isBeta
 ```
 
 ### With drilldown breadcrumbs
 
+Use [breadcrumbs](/components/breadcrumb) when a drilldown menu has more than 2 levels to offer users better navigation.
```

Comment:
It didn't seem like that example was scrollable right now, but it will be when the next build goes live? I added this and also moved some examples above the drilldown examples so that the isScrollable property is mentioned elsewhere first.

Reply:
Yep! The other PR got merged so this new description will be accurate